scripts/build-fontawesome.js

```diff
@@ -9,13 +9,13 @@ var fonts = [
 	"fontawesome-webfont.woff2"
 ];
 
-fs.ensureDir(destDir, function (err) {
-	if (err) {
-		console.error(err);
+fs.ensureDir(destDir, function(dirErr) {
+	if (dirErr) {
+		console.error(dirErr);
 	}
 
-	fonts.forEach(function (font) {
-		fs.copy(srcDir + font, destDir + font, function (err) {
+	fonts.forEach(function(font) {
+		fs.copy(srcDir + font, destDir + font, function(err) {
```

Comment:
Urgh, I know this is to be consistent with the rest of our code, but this is painful to look at 😅. Sorry I missed this.
Also, I believe I should have used `=>` functions there instead...